.infrastructure/deployment.yml

```diff
@@ -52,3 +52,20 @@ spec:
             port: 8080
           initialDelaySeconds: 5
           periodSeconds: 5
+        volumeMounts:
+          - name: todoapp-secrets-volume
+            mountPath: /app/secrets
+            readOnly: true
+          - name: todoapp-config-volume
+            mountPath: /app/configs
+            readOnly: true
+      volume:
+        - name: data
+          persistentVolumeClaim:
+            claimName: pvc-todoapp
+        - name: todoapp-secrets-volume
+          secret:
+            secretName: app-secret
+        - name: todoapp-config-volume
+          ConfigMap:
```

Comment:
The key 'ConfigMap' should be in lowercase ('configMap') to correctly reference a ConfigMap in the volumes section.